packages/react-sdk/src/components/components.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * This file should contain all the components that we use in the builder/production app
+ *
+ * The application imports only the components it uses and uses
+ * getComponent = createGetComponent({ Box, BlaBla })to pass them to RootInstance
+ * see example /packages/sdk-size-test/app/routes/$.tsx
+ */
+import { Body } from "./body";
+import { Box } from "./box";
+import { TextBlock } from "./text-block";
+import { Heading } from "./heading";
+import { Paragraph } from "./paragraph";
+import { Link } from "./link";
```

Comment:
why not just export it right away?

Reply:
Forgot thank you, initially tried do everything in single file